### ISSUE TITLE
Optic nerve sheath dilatation

### ISSUE THREAD

**Issue by daboe01** [obophenotype/human-phenotype-ontology#11102]
Preferred term label:
Optic nerve sheath dilatation

Synonyms
Dilation of the optic nerve sheath, Enlarged optic nerve sheath, Optic nerve sheath expansion

Definition (free text, please give PubMed ID)
An abnormal increase in the diameter of the optic nerve sheath, typically due to elevated intracranial pressure or local pathology affecting cerebrospinal fluid (CSF) dynamics. It can be detected via imaging modalities such as ultrasound or MRI and may be associated with papilledema, idiopathic intracranial hypertension, or optic nerve tumors.

Parent term (use hpo.jax.org/app)
Abnormal optic nerve morphology (HP:0000587)

Diseases characterized by this term
Idiopathic intracranial hypertension, Optic nerve glioma, Neurofibromatosis type 1

Your nano-attribution (ORCID)
0000-0001-9042-0508

**Comment by pnrobinson** [comment]
@daboe01 
Adding new term: 
- ID: HP:6001281
- label: Optic nerve sheath dilatation
- parents: [Abnormal optic nerve morphology (HP:0000587)](https://hpo.jax.org/app/browse/term/HP:0000587)
- pmid: PMID:39585390
- def: An abnormal increase in the diameter of the optic nerve sheath, typically due to elevated intracranial pressure or local pathology affecting cerebrospinal fluid (CSF) dynamics. It can be detected via imaging modalities such as ultrasound or MRI and may be associated with papilledema, idiopathic intracranial hypertension, or optic nerve tumors.
- comment: 
- orcid: orcid:0000-0001-9042-0508
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11102
- synonyms: Dilation of the optic nerve sheath(NONE)|Enlarged optic nerve sheath(NONE)|Optic nerve sheath expansion(NONE)
